data/tokens.json

```diff
@@ -2531,56 +2733,6 @@
       }
     }
   },
-  "badge": {
-    "gtmds_badge_height-xs": {
-      "value": "{gtmds_size-md}",
-      "type": "sizing"
-    },
-    "gtmds_badge_height-sm": {
-      "value": "{gtmds_size-lg}",
-      "type": "sizing"
-    },
-    "gtmds_badge_height-md": {
-      "value": "{gtmds_size-xl}",
-      "type": "sizing"
-    },
-    "gtmds_badge_height-lg": {
-      "value": "{gtmds_size-xl}+{gtmds_size-md}",
-      "type": "sizing"
-    },
-    "gtmds_badge_background-color-approval": {
-      "value": "{gtmds_color_purple-8}",
-      "type": "color"
-    },
-    "gtmds_badge_background-color-notification": {
-      "value": "{gtmds_color_red-8}",
-      "type": "color"
-    },
-    "gtmds_badge_background-color-draft": {
-      "value": "{gtmds_color_gray-6}",
-      "type": "color"
-    },
-    "gtmds_badge-label_color": {
-      "value": "{gtmds_color_white}",
-      "type": "color"
-    },
-    "gtmds_badge_border-radius": {
-      "value": "{gtmds_border-radius-max}",
-      "type": "borderRadius"
-    },
-    "gtmds_badge-label_typography-sm": {
-      "value": "{paragraph_sm-accented}",
-      "type": "typography"
-    },
-    "gtmds_badge-label_typography-md": {
-      "value": "{paragraph_md-accented}",
-      "type": "typography"
-    },
-    "gtmds_badge-label_typography-lg": {
-      "value": "{paragraph_lg-accented}",
-      "type": "typography"
-    }
-  },
   "notifications": {
```

Comment:
Why revamp here?

Reply:
this is the badge thing I tagged you for. i just moved it in the plugin UI to where it should be alphabetically